hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/Invocation.java

```diff
@@ -419,11 +419,10 @@ void notifyNormalResponse(Object value, int expectedBackups) {
     @SuppressFBWarnings(value = "VO_VOLATILE_INCREMENT",
             justification = "We have the guarantee that only a single thread at any given time can change the volatile field")
     void notifyCallTimeout() {
-        if (context.logger.isFinestEnabled()) {
-            context.logger.finest("Call timed-out either in operation queue or during wait-notify phase, retrying call: " + this);
-        }
-
         if (!(op instanceof BlockingOperation)) {
+            if (context.logger.isFinestEnabled()) {
+                context.logger.finest("Call timed-out either in operation queue or during wait-notify phase, retrying call: " + this);
```

Comment:
Says `Call timed-out either in operation queue or during wait-notify phase, retrying call..`, but here it's not retried.